pom.xml

```diff
@@ -7,13 +7,12 @@
 	<name>Calico Home Security</name>
 	<build>
 		<sourceDirectory>src/module_PI/Raspberry_PI</sourceDirectory>
-		<!--<testSourceDirectory>src/module_PI/Raspberry_PI/JUnitTests</testSourceDirectory>--> <!-- TODO: Adjust tests so they don't rely on an Arduino being plugged in (should be mocked) and running then uncomment this line so that all unit tests are run at build time -->
+		<testSourceDirectory>src/module_PI/Raspberry_PI/JUnitTests</testSourceDirectory> <!-- TODO: Adjust tests so they don't rely on an Arduino being plugged in 
```

Comment:
No, I just ran the tests with the Arduino plugged in. I should read about that mocked arduino. I'm just trying to prioritize